modules/cli/src/main/java/org/apache/ignite/cli/deprecated/spec/NodeCommandSpec.java

```diff
@@ -102,12 +130,32 @@ public Integer call() {
             out.println(tbl);
             return 0;
         }
+
+        private Path getConfigPath() {
+            return configOptions != null ? configOptions.configPath : null;
+        }
+
+        private String getConfigStr() {
```

Comment:
Well, looks like these parameters (port, rest-port etc) cannot be provided with a configuration file. Right?
It seems to me, that ability to start two nodes with the same configuration file and overridden value of `port` would be very convenient. I mean the following:
`node start nodeName --config \path_to_configuration_file --port 12345`

What do you think?

Reply:
Yes I think this is possible and will be helpful.